src/views/Login.vue

```diff
@@ -48,11 +53,21 @@ export default defineComponent({
   data() {
     return {
       username: "",
-      password: ""
+      password: "",
+      instanceUrl: ""
     };
   },
+  computed: {
+    ...mapGetters({
+      instanceUrlSaved: 'user/getInstanceUrl'
+    })
+  },
+  mounted() {
```

Comment:
Why we have used mounted hook here? I think we can directly use the getter like `this.instanceUrlSaved` in the template.